### DISCUSSION TITLE
Next Step after Colab Training

### DISCUSSION THREAD

**Discussion by Agha-Muqarib** [lukas-blecher/LaTeX-OCR#90] [Q&A] [answered]
Greetings @lukas-blecher, Hoping you're doing good!

I trained the Model on Google Colab Notebook you provided. I wanted to ask, how to download the trained project on my Local Machine. What is the next step? The objective is to be able to generate LaTeX from handwritten equations. Looking forward to hearing from you!

**Comment by lukas-blecher** [comment, marked as answer]
The weights are regularly saved. You can download them by locating them in the file browser. They are saved at /content/LaTeX-OCR/checkpoints/mixed

**Reply by dungcan2k5** [reply]
Thank you so much for your answer, it's really what I've been looking for for a long time

**Comment by SaadAliHafiz** [comment]
I've downloaded the weight.pth file and placed into checkpoints folder. How do I check if it's working fine?
I've tried checking it by taking snip of handwritten equations but it's not giving me accurate results.

**Reply by lukas-blecher** [reply]
Evaluate it on the test set
```
python eval.py -d path/to/crohme_test.pkl -c path/to/latest/checkpoint --config path/to/config.yaml
```
If that's not returning satisfactory results you need further training and probably more data, but that's a bit tricky.

**Reply by SaadAliHafiz** [reply]
Okay thanks!
